server/org_config.go

```diff
@@ -0,0 +1,182 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/influxdata/chronograf"
+)
+
+type organizationConfigResponse struct {
+	Links selfLinks `json:"links"`
+	chronograf.OrganizationConfig
+}
+
+func newOrganizationConfigResponse(config chronograf.OrganizationConfig) *organizationConfigResponse {
+	return &organizationConfigResponse{
+		Links: selfLinks{
+			Self: "/chronograf/v1/org_config",
+		},
+		OrganizationConfig: config,
+	}
+}
+
+type logViewerConfigResponse struct {
+	Links selfLinks `json:"links"`
+	chronograf.LogViewerConfig
+}
+
+func newLogViewerConfigResponse(lv chronograf.LogViewerConfig) *logViewerConfigResponse {
+	return &logViewerConfigResponse{
+		Links: selfLinks{
+			Self: "/chronograf/v1/org_config/logviewer",
+		},
+		LogViewerConfig: lv,
+	}
+}
+
+// OrganizationConfig retrieves the organization-wide config settings
+func (s *Service) OrganizationConfig(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	orgID, ok := hasOrganizationContext(ctx)
+	if !ok {
+		Error(w, http.StatusBadRequest, "Organization not found on context", s.Logger)
+		return
+	}
+
+	config, err := s.Store.OrganizationConfig(ctx).FindOrCreate(ctx, orgID)
+	if err != nil {
+		Error(w, http.StatusBadRequest, err.Error(), s.Logger)
+		return
+	}
+
+	if config == nil {
+		Error(w, http.StatusBadRequest, "Organization configuration object was nil", s.Logger)
+		return
+	}
+	res := newOrganizationConfigResponse(*config)
+
+	encodeJSON(w, http.StatusOK, res, s.Logger)
+}
+
+// LogViewerOrganizationConfig retrieves the log viewer UI section of the organization config
```

Comment:
Comment should explain why we do a find or create

Reply:
👍

Reply:
good call